docs/en/enterprise-edition/policy-reference/azure-policies/azure-general-policies/bc-azure-237.adoc

```diff
@@ -0,0 +1,45 @@
+
+== Dedicated data endpoint is disabled
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|===
+|Prisma Cloud Policy ID
+| TBD
+
+|Checkov ID
+| https://github.com/bridgecrewio/checkov/blob/main/checkov/terraform/checks/resource/azure/ACRDedicatedDataEndpointEnabled.py[CKV_AZURE_237]
+
+|Severity
+|LOW
+
+|Subtype
+|Build
+
+|Frameworks
+|Terraform,TerraformPlan
+
+|===
+
+=== Description
+
+This policy is checking to ensure that dedicated data endpoints are enabled in Azure. Dedicated data endpoints represent an enhanced level of security in a cloud environment by ensuring that data connections occur directly through private IP addresses within your virtual network. If dedicated data endpoints are not enabled, it could pose a security risk by allowing data to traverse over the public internet, increasing the risk of data interception or breaches.
```

Comment:
This policy checks whether dedicated data endpoints are enables in Azure. Dedicated data endpoints enhance security by directing data connections through private IPs within your virtual network. Disabled endpoints expose data to the public internet, increasing the risk of interception or breaches. Enabling dedicated data endpoints strengthens your security posture.